<p>Any @import rules must precede all other at-rules and style rules in a style sheet (besides @charset, which must be the first thing in the style sheet if it exists), or else the @import rule is invalid.</p>

<h2>Noncompliant Code Example</h2>
<pre>
@charset "UTF-8";

.mybox {
  color: green;
}

@import "mystyle.css"; /* Noncompliant: .mybox rule precedes the @import rule */
</pre>

<h2>Compliant Solution</h2>
<pre>
@charset "UTF-8";

@import "mystyle.css";

.mybox {
  color: green;
}
</pre>

<h2>See</h2>
<ul>
    <li><a href="https://www.w3.org/TR/css3-cascade/#at-ruledef-import">W3C Specifications</a></li>
    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@import">Mozilla Developer Network @import Documentation</a></li>
</ul>
